google/iac/master-compliance-test.json

```diff
@@ -5007,6 +5007,78 @@
                         "gcp-types/logging-v2:projects.metrics"
                     ]
                 },
+                {
+                    "masterTestId": "PR-AWS-0001-RGX",
+                    "masterSnapshotId": [
+                        "GDF_TEMPLATE_SNAPSHOT"
```

Comment:
@ishan-pansuriya please change `aws` to `gcp`
also do it for all other IDs as well